egs/hub4_spanish/s5/local/arpa2G.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+# Copyright 2013-2014  Johns Hopkins University (authors: Yenda Trmal, Daniel Povey)
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+# WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+# MERCHANTABLITY OR NON-INFRINGEMENT.
+# See the Apache 2 License for the specific language governing permissions and
+# limitations under the License.
+
+#Simple utility script to convert the gzipped ARPA lm into a G.fst file
```

Comment:
Is it necessary to use the "local" arpa2G script?  Does it do anything that the one in utils/ doesn't do, that we need?